app/controllers/admin/routes.py

```diff
@@ -38,6 +38,25 @@
 from app.logic.serviceLearningCourses import parseUploadedFile, saveCourseParticipantsToDatabase, unapprovedCourses, approvedCourses, getImportedCourses, getInstructorCourses, editImportedCourses
 
 from app.controllers.admin import admin_bp
+from flask import send_file, render_template, request
+from app.controllers.admin import admin_bp
+from app.logic.spreadsheet import createSpreadsheet
+
+
+@admin_bp.route('/admin/reports')
+def reports():
+    academic_years = Term.select(Term.academicYear).distinct().order_by(Term.academicYear.desc())
+    academic_years = list(map(lambda t: t.academicYear, academic_years))
+    return render_template("/admin/reports.html", academic_years=academic_years)
```

Comment:
Our variables are camelCase in our application

Reply:
Changed names to camelCase.